base/aws/kustomization.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
 - github.com/utilitywarehouse/thanos-manifests/base/local?ref=v1.2.3
+- github.com/utilitywarehouse/thanos-manifests/base/global?ref=v1.2.3
```

Comment:
I don't understand why these are versioned URIs and not simply paths within the repository.

Reply:
@george-angel agreed this is unnecessary - only added for consistency. Will remove both :+1: